README.md

```diff
@@ -2,72 +2,84 @@
+    * Movies opening soon - Enter your zip code and receive a list of movies and their details.
+    * Libraries near you -  Enter your zip code and receive a list of libraries and their details.
+    * Programming meetups near you - Choose from an events list and receive details.
+    * News reader - List articles and read an article of your choosing.
+4. Your CLI application should not be too similiar to the Ruby final projects (Music Library CLI, Tic-Tac-Toe with AI, Student Scraper). Also, please refrain from using [Kickstarter](https://learn.co/tracks/full-stack-web-development-v5/object-oriented-ruby/scraping/kickstarter-scraping-lab) as that was used for the scraping 'code along'. 
+5. Use good OO design patterns. You should be creating a collection of objects, not hashes, to store your data. *Pro Tip: Avoid scraping data more than once per web page - utilize objects you have already created. It will speed up your program!*
 
+[World's Best Restaurants](https://github.com/cjbrock/worlds-best-restaurants-cli-gem) was built by a Learn student, is maintained by Flatiron School staff, meets these requirements, and is well coded. Take the time to clone, run, and look through the code in detail - it'll really help.
 
-*For bonus points, instead of just creating an application, create a gem and for extra bonus points try publishing it to RubyGems.*
+*If you have time for an extra challenge, [create a gem](https://guides.rubygems.org/make-your-own-gem/) from your application. Want more? Try [publishing your gem to RubyGems.org](https://guides.rubygems.org/publishing/).*
 
-### Example Domains
 
-- Movies opening soon - Enter your zip code and receive a list of movies and their details.
-- Libraries near you -  Enter your zip code and receive a list of libraries and their details.
-- Programming meetups near you, list details.
-- News reader - List articles, read article.
+## Instructions
 
-[now-playing](https://github.com/learn-co-curriculum/now-playing-cli-gem) is an example of a gem that would meet these requirements.
-[worlds best restaurants](https://github.com/cjbrock/worlds-best-restaurants-cli-gem) was built by a Learn student, maintained by Flatiron School staff, meets these requirements, and is well coded.
-Take the time to clone, run and look through the code for both of these in detail - it'll really help.
+1. Watch this [video walkthrough](https://www.youtube.com/watch?v=_lDExWIhYKI) of building a basic CLI Gem called [Daily Deal](https://github.com/learn-co-curriculum/daily_deal) before you begin. Review the [Student Scraper Project](https://learn.co/tracks/full-stack-web-development-v5/object-oriented-ruby/final-projects/student-scraper), which provides a good example of how scraping concerns can be separated into a distinct class.
 
-## Instructions
+2. Build your application. This is going to be different from labs you have done on Learn. Do *not* click `Open IDE` and create your application within the lesson files. Instead:
```

Comment:
I wonder if we can 'test drive' these instructions with any students before we deploy, maybe someone in the beta program?